notebooks/MAST/TESS/beginner_tour_lc_tp/requirements.txt

```diff
@@ -0,0 +1,3 @@
+astropy==3.1.dev22604
+matplotlib==2.2.2
+numpy==1.15.1
```

Comment:
Sorry I didn't catch this earlier Susan. (This fix does not involve touching your notebook though).

These should just be the lowest "known good" version and should not include dev versions. So if you could change the "==" to ">=" and the astropy version to 3.0.4 (I tested on that version so I know it works), that would be awesome.